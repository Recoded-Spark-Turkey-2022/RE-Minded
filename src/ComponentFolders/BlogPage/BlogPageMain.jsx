import React from "react";
import Top from "./MainContent";
import Bottom from "./RecommendedforYou";
import SignUp from "./SignUp";

function Main (){
    return(
        <>
        <Top /> 
        <SignUp />
        <Bottom />
        </>
    );
}

export default Main;