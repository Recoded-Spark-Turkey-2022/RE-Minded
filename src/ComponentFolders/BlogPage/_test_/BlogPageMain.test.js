import React from "react";
import  renderer  from "react-test-renderer";
import BlogPageMain from "../BlogPageMain"



test ( "BlogPageMain test snapshot", () => { 
    const component=renderer.create(<BlogPageMain />)
    let tree= component.toJSON(); 
    expect(tree).toMatchSnapshot(); 
})