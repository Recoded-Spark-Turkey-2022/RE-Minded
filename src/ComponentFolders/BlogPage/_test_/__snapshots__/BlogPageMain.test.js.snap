// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPageMain test snapshot 1`] = `
Array [
  <div
    className=" inline-block place-content-center "
  >
    <img
      alt="blog"
      className=" mx-40 mt-14 md:h-2/5 md:w-2/3 h-2/5 w-2/3"
      src="TopImage.svg"
    />
    <h1
      className=" mt-10 mx-40 font-semibold font-sans text-start uppercase md:text-2xl text-xl"
    >
      Mental Health and Technology
    </h1>
    <p
      className=" mt-4 text-start md:text-sm text-xs mx-40 font-sans uppercase"
    >
      Technology and mental well-being might seem like profoundly antagonistic terms. However, there are excellent digital tools to help you reach mindfulness and to practice meditation. You can even use smartphone apps, virtual reality solutions and digital devices as a form of relieving stressing and achieving digital detox – as, after a while, you will be able to practice all types of anti-anxiety skills on your own. So, let The Medical Futurist show you how you can enhance your mental and emotional health with the help of technology.
    </p>
    <h2
      className=" mt-10 mx-40 font-semibold font-sans uppercase text-start md:text-2xl text-xl"
    >
       The need for taking care of our emotional health 
    </h2>
    <p
      className="mt-4 text-start md:text-sm text-xs mx-40 font-sans uppercase"
    >
      “Just as we observe physical hygiene to stay well, we need to cultivate a kind of emotional hygiene too,” wrote the Dalai Lama on his Twitter-feed. There are three crucial points around His Holiness’ statement. First and foremost, the popularity of health-enhancing, physically active lifestyle is soaring worldwide. Nevertheless, there’s always room for improvement: the WHO says that globally, 23 percent of adults and 81 percent of school-going adolescents are not active enough. According to the organization, adults aged 18-64 should do at least 150 minutes of moderately intense physical activity each week, which is not unaccomplishable. Moreover, the numerically and statistically driven 21st-century mind could take a great inspiration and power from the various fitness trackers and wearables. Secondly, emotional hygiene seems to be thrown into the shade nowadays. Somehow, the culture of busyness prevents us from squeezing some time for mental health into our packed schedules. Information is pouring on us from millions of communication channels; we are connecting with hundreds of ‘friends’ on social media, we are obsessed with data due to the fear of missing out and time management slices up our days into meetings and to-do-lists. The pressure to perform and the stress to achieve is the air every busy-bee is breathing. However, turbulent lifestyle comes with a price. No wonder so many people have mental health issues. In the US, that means one in five adults – more than 40 million Americans! On the old continent, 27 percent of the population has to cope with mental disorders, which means 83 million struggling Europeans. In Asia, the prevalence of depression is 20 percent in Thailand, 19.9 percent in Taiwan, 19.4 percent in Korea, 17.5 in Malaysia and 16.5 percent in China.
    </p>
  </div>,
  <div
    className=" inline-block items-center "
  >
    <h2
      className=" mt-10 mx-40 font-semibold font-sans text-start uppercase md:text-2xl text-xl"
    >
      Sign up for The Healing blog
    </h2>
    <p
      className=" mt-2 text-start md:text-sm text-xs uppercase mx-40"
    >
      A weekly, ad-free Blog that helps you stay in the know.
    </p>
    <div
      className="md:flex"
    >
      <div
        className=" mt-7 mx-40 flex justify-start mb-4 w-[300px] h-[50px] box-border rounded-md border-2 border-[#718096]"
      >
        <input
          className=" w-3/4 h-auto rounded-l-lg text-SubTexts font-normal "
          placeholder="  Enter your e-mail"
          type="email"
        />
        <button
          className="bg-Buttons bg-sky-500 rounded-r-lg w-1/4 h-auto"
          type="button"
        >
          <img
            alt="Arrow"
            className="w-[24px] h-[24px] top-[18px] left-[310px] items-center mx-auto"
            src="SubscribeArrow.svg"
          />
        </button>
      </div>
    </div>
  </div>,
  <div>
    <div
      className="  inline-block  mt-32 mx-40 "
    >
      <h2
        className=" font-semibold font-sans text-start uppercase md:text-2xl text-xl"
      >
        Recommended for you
      </h2>
      <div
        className=" flex flex-col md:flex-col lg:flex-row mt-10"
      >
        <img
          alt="Heal"
          className=" mr-10 h-64 w-96"
          src="BottomImage1.svg"
        />
        <img
          alt="Need"
          className=" h-64 w-96"
          src="BottomImage2.svg"
        />
      </div>
    </div>
  </div>,
]
`;
